cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/aggregator/KafkaPartitionMetricSampleAggregatorTest.java

```diff
@@ -321,12 +322,12 @@ public void testExcludeInvalidMetricSample() throws NotEnoughValidWindowsExcepti
     ValuesAndExtrapolations partitionValuesAndExtrapolations = valuesAndExtrapolations.get(PE);
     for (Resource resource : Resource.cachedValues()) {
       Collection<Short> metricIds = KafkaMetricDef.resourceToMetricIds(resource);
-      double expectedValue = (resource == Resource.DISK ?
-          MIN_SAMPLES_PER_WINDOW - 1 : (MIN_SAMPLES_PER_WINDOW - 1) / 2.0) * metricIds.size();
+      double expectedValue = (resource == Resource.DISK ? MIN_SAMPLES_PER_WINDOW - 1 : (MIN_SAMPLES_PER_WINDOW - 1) / 2.0)
+                             * (resource == Resource.CPU ? 0.01 : 1.0) * metricIds.size();
```

Comment:
Nit: Moving these hard coded values to static variables would make the code more clear.